.env.example

```diff
@@ -16,6 +16,4 @@ NODE_VERSION=X-Node-Version
 
 NODE_ENV = development
 
-
-
-
+JEEK_JOKES_API_URL=https://example.jokes.com/api/v1
```

Comment:
This file is not necessary. It's just an example. You don't have to update it.
In fact, you can delete it if you want.